test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompaction_2_IT.java

```diff
@@ -238,9 +238,6 @@ public void testDeleteTableCancelsUserExternalCompaction() throws Exception {
       confirmCompactionCompleted(getCluster().getServerContext(), ecids,
           TCompactionState.CANCELLED);
 
-      // ELASTICITY_TODO make delete table fate op get operation ids before deleting
-      // there should be no metadata for the table, check to see if the compaction wrote anything
-      // after table delete
       try (var scanner = client.createScanner(AccumuloTable.METADATA.tableName())) {
```

Comment:
Could take a bit of the comment that describes the purpose of the scan

```suggestion
      // Ensure compaction did not write anything to metadata table after delete table
      try (var scanner = client.createScanner(AccumuloTable.METADATA.tableName())) {
```